common/sky/events.py

```diff
@@ -14,9 +14,12 @@
 
 
 def night_events(time: Time, location: EarthLocation, localtzone: timezone) -> \
-        Tuple[Time, Union[npt.NDArray[float], Time], Union[npt.NDArray[float], Time], Union[npt.NDArray[float], Time],
-              Union[npt.NDArray[float], Time], Union[npt.NDArray[float], Time],
-              Union[npt.NDArray[float], Tuple[Angle, Time, EarthLocation]]]:
+        Tuple[Time, Union[npt.NDArray[float], Time],
+              Union[npt.NDArray[float], Time],
+              Union[npt.NDArray[float], Time],
+              Union[npt.NDArray[float], Time],
+              Union[npt.NDArray[float], Time],
+              Union[npt.NDArray[float], Time]]:
```

Comment:
We might need an alias for this?

Reply:
LOL probably a good idea. I'll add that.